aliases/available/tmux.aliases.bash

```diff
@@ -0,0 +1,10 @@
+cite 'about-alias'
+about-alias 'Tmux terminal multiplexer'
+
+case $OSTYPE in
+  linux*)
```

Comment:
Any reason why these are specific to Linux? Shouldn't they work on other operating systems as well, as long as tmux is installed?